install/ui/test/README.md

```diff
@@ -1,18 +1,28 @@
 # WebUI Unit Tests
 
-How to run tests:
+Prerequisites:
```

Comment:
How about your turn this and the other section titles into a level 2 title ``## Prerequisites``?

Reply:
Yes, that definitely makes sense.